mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_Mobile.java

```diff
@@ -1253,6 +1278,16 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
                 mUserLocation.onRuntimePermissionsGranted();
             }
         }
+        if (requestCode == 355) {
```

Comment:
Maybe this code should be deleted?

Reply:
Maybe.. i will add an TODO
I have trying to add the file permission but there changed so many things with sdk 33